modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationModelIndexer.java

```diff
@@ -298,20 +315,10 @@ private void _addLocalizedText(
 		Document document, ResourceBundleLoader resourceBundleLoader,
 		List<TranslationHelper> translationHelpers) {
 
-		ResourceBundle defaultResourceBundle =
-			resourceBundleLoader.loadResourceBundle(LocaleUtil.getDefault());
-
 		for (Locale locale : LanguageUtil.getAvailableLocales()) {
-			ResourceBundle resourceBundle =
-				resourceBundleLoader.loadResourceBundle(locale);
-
 			for (TranslationHelper translationHelper : translationHelpers) {
-				if (resourceBundle != null) {
-					translationHelper.accept(resourceBundle, locale);
-				}
-				else if (defaultResourceBundle != null) {
-					translationHelper.accept(defaultResourceBundle, locale);
-				}
+				translationHelper.accept(
+					_getResourceBundle(locale, resourceBundleLoader), locale);
```

Comment:
I like the abstraction of getting the resource bundle but it looks like we're losing the null check on the default resource bundle.  Did I miss something?